pac4j-core/src/main/java/org/pac4j/core/context/J2EContext.java

```diff
@@ -202,6 +202,7 @@ public String getFullRequestURL() {
         if (cookies != null) {
             for (final javax.servlet.http.Cookie c : cookies) {
                 final Cookie cookie = new Cookie(c.getName(), c.getValue());
+                cookie.setVersion(c.getVersion());
```

Comment:
Is version 0 the default value? Just to be sure we don't break anything for those who use cookies.